src/app/page/admin/decks/deck/manageCards/admin.manage.cards.component.ts

```diff
@@ -14,21 +15,23 @@ import {CreateCardDTO} from '../../../../../dto/CardsDTO/CreateCardDTO';
 })
 
 export class AdminManageCardsComponent implements OnInit {
-  public cards: CreateCardDTO[] = [];
+  public blockEdit: boolean = true;
+  public cards: CardPublic[] = [];
   public deck: AdminDeck;
-  public card: CreateCardDTO;
+  public card: CardPublic;
   public question: string;
   public answer: string;
   public title: string;
   public rating: '0';
   private url: string;
   private sub: Subscription;
+  public cardLink: string;
```

Comment:
I would keep _cardLink_ type as _Link_ and convert to _string_ only when used, that means in method updateCard().

Reply:
Ok, i will fix it.